app/controllers/serviceLearning/routes.py

```diff
@@ -208,6 +228,55 @@ def renewCourse(courseID, termID):
 
     return "", 500
 
+@serviceLearning_bp.route('/serviceLearning/print/<courseID>', methods=['GET'])
+def printCourse(courseID):
+    """
+    This function will print a PDF of an SLC proposal.
+    """
+    instructors = CourseInstructor.select().where(CourseInstructor.course==courseID)
+    courseInstructors = [instructor.user for instructor in instructors]
+    isCreator = Course.select(Course.createdBy.username).join(User).where(Course.createdBy.username == g.current_user, Course.id==courseID).exists()
```

Comment:
this query can be a little simpler and doesn't require the join to user: `Course.select().where(Course.createdBy == g.current_user, Course.id==courseID).exists()`